samtranslator/internal/schema_source/aws_serverless_function.py

```diff
@@ -405,6 +405,7 @@ class MSKEvent(BaseModel):
 class MQEventProperties(BaseModel):
     BatchSize: Optional[PassThroughProp] = mqeventproperties("BatchSize")
     Broker: PassThroughProp = mqeventproperties("Broker")
+    DynamicPolicyName: Optional[bool]  # TODO: add docs
```

Comment:
Resolve the TODO?

Reply:
Discussed it with @hoffa We add docs when it's in released and deployed everywhere. After that I'll resolve it

Reply:
cannot until the doc is out

Reply:
We only add it once https://github.com/awsdocs/aws-sam-developer-guide is updated (i.e. the change has deployed).